rfc6962-bis.xml

```diff
@@ -1201,6 +1201,9 @@ an accepted root certificate.
                     <t hangText="extra_data:">
                       The base64-encoded unsigned data pertaining to the log entry. In the case of an X509ChainEntry, this is the <spanx style="verb">certificate_chain</spanx>. In the case of a PrecertChainEntryV2, this is the whole <spanx style="verb">PrecertChainEntryV2</spanx>.
                     </t>
+                    <t hangText="signed_certificate_timestamp:">
+                      A Signed Certificate Timestamp for the entry, in the same format as in <xref target="sct"/>.
```

Comment:
As mentioned elsewhere, I suspect add-(pre-)chain should also return an SCT blob.